grpc_opa/authorizer.go

```diff
@@ -293,7 +331,7 @@ func (a *DefaultAuthorizer) AffirmAuthorization(ctx context.Context, fullMethod
 
 	ok, newCtx, err = a.Evaluate(ctx, fullMethod, grpcReq, a.OpaQuery)
 	if err != nil {
-		logger.WithError(err).Errorf("unable_authorize %#v", a)
+		logger.WithError(err).Errorf("unable_authorize %s", a)
```

Comment:
Use structured logs here
```suggestion
		logger.WithError(err).WithField("authorizer", a).Error("unable_authorize")
```